app/views/job_templates/run_openscap_remediation_-_ansible_default.erb

```diff
@@ -22,7 +22,14 @@ require:
 ---
 - hosts: all
   tasks:
-<%= indent(4) { input('tasks') } -%>
```

Comment:
This should be placed after the bootc stuff, right?

Reply:
All set now.